e2e-polybft/property/property_test.go

```diff
@@ -55,3 +56,74 @@ func TestProperty_DifferentVotingPower(t *testing.T) {
 		require.NoError(t, cluster.WaitForBlock(numBlocks, blockTime*time.Duration(numBlocks)))
 	})
 }
+
+func TestProperty_DropValidators(t *testing.T) {
+	t.Parallel()
```

Comment:
I'd remove parallel execution, because it might reuse the same rootchain server when executing tests. So far we had only one test, but now when we have 2 of them, we witness these issues (replacing transactions, etc.). Definetly worth trying.
```suggestion
```